api/tests/e2e/routes_test.go

```diff
@@ -243,11 +244,13 @@ var _ = Describe("Routes", func() {
 			It("returns success and routes the host to the app", func() {
 				Expect(resp).To(HaveRestyStatusCode(http.StatusOK))
 
-				appClient := resty.New()
+				appClient := resty.New().SetTLSClientConfig(&tls.Config{InsecureSkipVerify: true})
 				Eventually(func() int {
 					var err error
-					resp, err = appClient.R().Get(fmt.Sprintf("http://%s.%s", host, appFQDN))
-					Expect(err).NotTo(HaveOccurred())
+					resp, err = appClient.R().Get(fmt.Sprintf("https://%s.%s", host, appFQDN))
+					if err != nil {
+						return 0
+					}
```

Comment:
Returning 0 as an invalid http status is a bit weird to me. I would do one of the following instead:
- change the function signature to return (int, error) and simply propagate the error. Ginkgo will detect if the second argument is non-nil and fail if it is so.
- change the function signature to accept a gomege.Gomega arg which allows the use of the Succeed mathcher and inline assertions as explained in [the inline docs](https://github.com/onsi/gomega/blob/c8ba5823d8c284a7f8c070cb0f3ec3faf076db04/gomega_dsl.go#L292-L301)
I would lean towards the latter in this case since it will get rid of the if and make the code easier to read